fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeType.java

```diff
@@ -202,4 +206,42 @@ public static Map<String, Action> actionMap() {
             return ret;
         }
     }
+
+    public enum MaterializedViewAction {
+        ALTER(1),
+        REFRESH(2),
+        DROP(3);
```

Comment:
we also need to support `SELECT` for mv in new privilege framework, plz refer to this prd to make sure all the required privilege have been supported properly and upgraded properly. https://starrocks.feishu.cn/docx/FHCHdcSrLo08Dpx9LxtcplPkneg